myhpom/models/user.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.contrib.auth.models import User
+from myhpom import validators
+
+
+def user_pre_save_receiver(sender, instance, **kwargs):
+    validators.name_validator(instance.first_name)
```

Comment:
Document the purpose of this signal handler either by changing the method name to reflect its purpose, or with a docstring - I assume its ultimately to populate the username field, but I'm not sure.

Reply:
For future maintainers it would be very helpful to have a test_user_model.py that captures the intended use of this signal handler.

Reply:
Good idea – I've done both